cloud/scope/powervs_machine.go

```diff
@@ -1060,6 +1060,25 @@ func (m *PowerVSMachineScope) CreateVPCLoadBalancerPoolMember() (*vpcv1.LoadBala
 					}
 				}
 			}
+
+			for _, al := range lb.AdditionalListeners {
+				var shouldAdd bool
+
+				switch al.Enabled {
+				case infrav1beta2.AdditionalListenerAll:
+					shouldAdd = true
+				case infrav1beta2.AdditionalListenerBootstrapOnly:
+					shouldAdd = strings.Contains(m.IBMPowerVSMachine.Name, "-bootstrap")
+				case infrav1beta2.AdditionalListenerMastersOnly:
+					shouldAdd = strings.Contains(m.IBMPowerVSMachine.Name, "-master")
```

Comment:
I am not sure need to check but I think rather than name we can use labels/anotations which identify that a machine is control-plan or data?